library/src/main/java/com/testproject/library/ScreenUtils.kt

```diff
@@ -0,0 +1,34 @@
+package com.testproject.library
+
+import android.content.Context
+import android.util.DisplayMetrics
+import android.view.WindowManager
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/errysuprayogi/ReviewMe/blob/8e177c6791b29597c6309ee6405dc2c695977e74/library/src/main/java/com/testproject/library/ScreenUtils.kt#L5"></span>There should be exactly one empty line in between the list of imports and the declaration of ScreenUtils.</td>
    </tr>
  </tbody>
</table>